samples/snippets/quickstart_analyzeiampolicy.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+
+from google.cloud import asset_v1
+
+
+def build_analysis_query(parent):
+    analysis_query = asset_v1.IamPolicyAnalysisQuery()
+    analysis_query.scope = parent
+    analysis_query.resource_selector.full_resource_name = "//cloudresourcemanager.googleapis.com/" + parent
+    analysis_query.options.expand_groups = True
+    analysis_query.options.output_group_edges = True
+
+    return analysis_query
+
+
+def analyze_iam_policy(project_id):
+    # [START asset_quickstart_analyze_iam_policy]
+
+    # TODO project_id = 'Your Google Cloud Project ID'
+
+    client = asset_v1.AssetServiceClient()
+    parent = "projects/{}".format(project_id)
+    analysis_query = build_analysis_query(parent)
```

Comment:
Sorry for the late comment.

A helper function `build_analysis_query` is used from the sample code, but that function is outside of the region tag. Is this intended? I would show everything.

Reply:
Hi Takashi, I am not sure what region tag are you referring to? Could you share a bit more specific information?

Reply:
Sure, our documentation site will include the sample code between `# [START asset_quickstart_analyze_iam_policy]` and `# [END asset_quickstart_analyze_iam_policy]`. So the readers won't see anything about `build_analysis_query` which makes our documentation page incomplete.

Reply:
Done, let me know if this is the correct way to tag the helper functions.